dotnet/src/Plugins/Plugins.MsGraph/CloudDrivePlugin.cs

```diff
@@ -47,9 +47,11 @@ public CloudDrivePlugin(ICloudDriveConnector connector, ILoggerFactory? loggerFa
         Stream fileContentStream = await this._connector.GetFileContentStreamAsync(filePath, cancellationToken).ConfigureAwait(false);
 
         using StreamReader sr = new(fileContentStream);
-        string content = await sr.ReadToEndAsync().ConfigureAwait(false);
-
-        return content;
+#if NET
+        return await sr.ReadToEndAsync(cancellationToken).ConfigureAwait(false);
+#else
+        return await sr.ReadToEndAsync().ConfigureAwait(false);
+#endif
```

Comment:
Following the WriteAsync change above, this sounds less verbose
```suggestion
return await sr.ReadToEndAsync(
#if NET
    cancellationToken
#endif
).ConfigureAwait(false);
```